scss/_detail.scss

```diff
@@ -0,0 +1,61 @@
+%title {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+%content {
+  color: $dark-grey;
+  font-size: 16px;
+}
+
+.detail {
+  display: flex;
+  width: 100%;
+  max-width: 600px;
+  padding: 16px;
```

Comment:
In wolox we sort properties by alphabetical order.

Reply:
Done 😃